rdr_service/.configs/db_config.json

```diff
@@ -1 +1 @@
-{"db_connection_string": "mysql+mysqldb://rdr:rdr!pwd@127.0.0.1/?charset=utf8",   "backup_db_connection_string": "mysql+mysqldb://rdr:rdr!pwd@127.0.0.1/?charset=utf8",   "unittest_connection_string": "mysql+mysqldb://root@127.0.0.1:9306?charset=utf8",  "db_password": "rdr!pwd",   "db_connection_name": "",   "db_user": "rdr",   "db_name": "rdr" }
+{"db_connection_string": "mysql+mysqldb://rdr:rdr!pwd@127.0.0.1/?charset=utf8",   "backup_db_connection_string": "mysql+mysqldb://rdr:rdr!pwd@127.0.0.1/?charset=utf8",   "unittest_connection_string": "mysql+mysqldb://root@127.0.0.1:9306",  "db_password": "rdr!pwd",   "db_connection_name": "",   "db_user": "rdr",   "db_name": "circle_test" }
```

Comment:
You should not change the database config. The app expects the schema to be named rdr.

Reply:
Added to gitignore.